data/PratishShrivastava.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Pratish Shrivastava",
+  "url": "https://github.com/PratishShrivastava",
+  "image": "https://github.com/PratishShrivastava.png",
+  "why": "To be healty for long period of time.",
+  "weekly": ["Walk daily", "Eat more Healty Food"],
```

Comment:
Hello please fix your typos of healthy (lines 5, 6, 7) & want (7) thank you!

Reply:
done